into/backends/sql_csv.py

```diff
@@ -1,19 +1,25 @@
 from __future__ import absolute_import, division, print_function
 
+import os
+import re
+import subprocess
+
+import sqlalchemy
+
+from multipledispatch import MDNotImplementedError
+
 from ..regex import RegexDispatcher
 from ..append import append
 from .csv import CSV
-from multipledispatch import MDNotImplementedError
-import os
-import datashape
-import sqlalchemy
-import subprocess
+from ..temp import Temp
+from ..into import into
```

Comment:
As above, I've been avoiding bringing `into` into internal code.  This breaks that pattern.  I'm not sure if it's a pattern worth keeping.